src/Components/Web.JS/src/Platform/Mono/MonoDebugger.ts

```diff
@@ -3,10 +3,17 @@ import { WebAssemblyResourceLoader } from '../WebAssemblyResourceLoader';
 const currentBrowserIsChrome = (window as any).chrome
   && navigator.userAgent.indexOf('Edge') < 0; // Edge pretends to be Chrome
 
+let isDebugging = true;
+
+window.addEventListener('load', () => {
+  const params = new URLSearchParams(window.location.search);
+  isDebugging = params.get('_blazor_debug') === 'true';
```

Comment:
Should we call this `blazor_debug=true`? We would want users to possibly use this if they were debugging inside chrome no? Also, should we include this in the message we print as part of the Shift + Alt + D: https://github.com/dotnet/aspnetcore/blob/648c15dbe90fb8c113f7c6b4adeb40d9e10494f6/src/Components/WebAssembly/DebugProxy/src/TargetPickerUi.cs#L38?

Reply:
> Should we call this blazor_debug=true? 

Do you mean remove the underscore at the start or set it to default to try?

Reply:
I added the warning in the console when the user presses the keyboard shortcut. It's a little bit more straightforward to implement it there.

Reply:
I was going to suggest doing both. We do want for users to be able to set this if they'd prefer using the browser. Using `_` feels like messing with private state.